src/new-components/badge/badge.module.css

```diff
@@ -2,7 +2,7 @@
     --reactist-badge-neutral-background: rgb(238, 238, 238);
     --reactist-badge-neutral-foreground: rgb(102, 102, 102);
 
-    --reactist-badge-positive-background: rgba(5, 133, 39, 0.1);
+    --reactist-badge-positive-background: rgb(224, 241, 228);
```

Comment:
This change preserves the same color. The thing is, having had the color specified with some transparency, it worked great when the background is white, but when the background is something else, the color changed in unwanted ways.

So I changed the color to be the one that appears when on white, but hard-coded and not relying on transparency-over-white-bg to get it right. Now, the color is the same regardless of the background on which the badge appears.